widgetry/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 
 [features]
 native-backend = ["glutin", "usvg/system-fonts", "usvg/text"]
-wasm-backend = ["instant/wasm-bindgen", "usvg/text", "wasm-bindgen", "web-sys", "winit/web-sys"]
+wasm-backend = ["instant/wasm-bindgen", "usvg/text", "wasm-bindgen", "web-sys"]
```

Comment:
This was just the fallout of updating winit - web-sys is now default so there's no feature flag for it.